### DISCUSSION TITLE
What version of Ghostscript is supported with ImageMagick_7.0.8.50 version

### DISCUSSION THREAD

**Discussion by manikcr24** [ImageMagick/ImageMagick#3472] [Development]
Hi Team,
Can someone please tell me what is the compatible Ghostscript version for ImageMagick_7.0.8.50? 

Thanks in advance,
Manikanta Cheruku.

**Comment by fmw42** [comment]
I suggest using the latest version of GS. I have heard reports about versions around 9.25-9.27 having bugs.

**Reply by manikcr24** [reply]
Hi Fred (@fmw42),
Thanks for responding to this thread. 
Actually, we are using GS_9.52 with IM_7.0.8.50. Our security team has identified some vulnerabilities in GS_9.52. So we tried upgrading to GS_ 9.53.3 (latest available version as of today). After the upgrade IM started throwing some errors. If we revert back to 9.52 then IM is working fine again.

**Reply by fmw42** [reply]
What about 9.51? Does that have the same vulnerabilities? Can you go back to last stable GS?

**Reply by manikcr24** [reply]
Hi Fred,
Yeah, and we wanted to upgrade gs to 9.53.3. So we wish to know if IM_7.0.8 supports gs_9.53.0 or above?

Thank you

**Reply by fmw42** [reply]
I know of no reason why it would not. But one of the IM developer would need to comment. The easiest thing would be just install and try it.

**Reply by dlemstra** [reply]
On which operating system are you experiencing this issue?

**Reply by manikcr24** [reply]
Hi @dlemstra,
We are facing this issue in :
OS Name:                   Microsoft Windows Server 2019 Standard
OS Version:                10.0.17763 N/A Build 17763

Thanks,
Manik.

**Reply by dlemstra** [reply]
I was suspecting already that you were running this on Windows. At some point they made a change that would make it impossible to run an older version of ImageMagick together with a new version of GhostScript. We added a patch for that on the 2020-09-15. So you will need a version of ImageMagick released after that date to make it work with a recent version of Ghostscript.

**Reply by manikcr24** [reply]
Hi @dlemstra,
Can you please point me to any resource(release note) that explains the compatibility of ImageMagick versions with Ghostscript versions?

Thank you very much.

**Reply by dlemstra** [reply]
Version `9.53` or higher of Ghostscript requires version `7.0.10-30` or higher of ImageMagick on Windows.

**Reply by manikcr24** [reply]
@dlemstra ,
Thank you very much for the confirmation.
